lldb/include/lldb/Utility/Status.h

```diff
@@ -170,12 +215,8 @@ class Status {
   bool Success() const;
 
 protected:
-  Status(llvm::Error error);
-  /// Status code as an integer value.
-  ValueType m_code = 0;
-  /// The type of the above error code.
-  lldb::ErrorType m_type = lldb::eErrorTypeInvalid;
-  /// A string representation of the error code.
+  Status(llvm::Error error) : m_error(std::move(error)) {}
+  mutable llvm::Error m_error;
```

Comment:
Any chance to avoid the mutable thing? This sort of means that accessing the object (even const methods) from multiple threads may not be thread-safe. I guess that sort of already was a problem with the m_string thingy, but I expect it to be more pronounced now.

Reply:
Done.